components/locid/src/extensions/transform/value.rs

```diff
@@ -45,8 +45,6 @@ impl Value {
     /// use icu::locid::extensions::transform::Value;
     ///
     /// let value = Value::try_from_bytes(b"hybrid").expect("Parsing failed.");
-    ///
-    /// assert_eq!(&value.to_string(), "hybrid");
```

Comment:
Observation: down here you just deleted the assertion. Probably fine to do this up above, too. Alternatively, you can use the macro on the single-subtag case.